app/gui2/src/App.vue

```diff
@@ -1,5 +1,9 @@
 <script setup lang="ts">
+import { onMounted } from 'vue'
+import { initializeMockDb } from '@/stores/suggestionDatabase'
```

Comment:
I had a problem with this approach, being a sum of the following:
1. The stores are initialized during first use
2. the first use of the store is when CB is first opened
3. The suggestionDB initialization is asynchronous
4. CB scrolls to the bottommost element only when opening or when input changes. Changes in the db itself does not change scrolling, as it could confuse users

Now, 1 and 2 and 3 => The CB list is empty during first opening, and when entries finally load, we do not select bottommost element due to 4.

And even in the eventual solution we should initialize db (i.e. get the state from LS and subscribe to updates) _before_ user opens CB, as initialization can take a bit of time.

I can of course just useSuggestionDbStore in App.vue, if it does any difference to you.

Reply:
Make initializeDatabase a method of the store, and useStore here.